src/components/frame/LandingPage/LandingPage.css

```diff
@@ -60,3 +62,14 @@
   composes: itemLink;
   text-align: center;
 }
+
+@media screen and query700 {
+  .main {
+    padding: 0 14px;
+    height: calc(100vh - 43px); /* @NOTE: 43px is the height of the nav */
```

Comment:
Could this be set using`@value` in `Default.css` and imported here? Similarly to how you import `query700` at the top

Reply:
Sure I can but just wondering why you would want to import `height: calc(100vh - 43px);` when its only used here compared to query700 which is used in many different places?